python/cuml/cuml/linear_model/logistic_regression_mg.pyx

```diff
@@ -336,7 +353,9 @@ class LogisticRegressionMG(MGFitMixin, LogisticRegression):
                         <int*> &num_iters)
 
                 else:
-                    assert self.index_dtype == np.int64, f"unsupported index dtype: {self.index_dtype}"
+                    assert self.index_dtype == np.int64, (
+                        f"unsupported index dtype: {self.index_dtype}"
+                    )
```

Comment:
Should we convert this as well? It sounds like it will break/do weird things if you use `int64`.

Same for the assert on L396 (old line count)